tools/travis/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+docker_image_prefix="testing"
+dockerhub_image_prefix="openwhisk"
+docker login -u "${DOCKER_USER}" -p "${DOCKER_PASS}"
+
+#capture couchdb setup
+docker commit couchdb "${docker_image_prefix}/couchdb"
+
+#deploy specific openwhisk core images
+for container in "controller"     \
+                 "invoker"        \
+                 "couchdb"        \
+                 "dockerskeleton" \
+                 "nodejs6action"  \
+                 "python2action"  \
+                 "python3action"  \
+                 "swift3action"   \
```

Comment:
i'd like to have a base "swift" image which swift3action builds from - the base image likely changes less often and would make the swift container build seconds vs minutes.